projects/addon-doc/components/navigation/navigation.template.html

```diff
@@ -1,19 +1,25 @@
-<tui-input
-    #input
+<tui-textfield
```

Comment:
@waterplea after you replaced `tui-input` with `tui-textfield`, the focus by keyboard shortcut might be broken since you haven't updated the `@ViewChild(TuiInputComponent)` query.